python_on_whales/components/pod/cli_wrapper.py

```diff
@@ -0,0 +1,778 @@
+        full_cmd.add_simple_arg("--time", time)
+        full_cmd.extend([str(p) for p in pods])
+        run(full_cmd)
+
+    def restart(
+        self,
+        x: Union[ValidPod, Sequence[ValidPod]] = (),
+        /,
+        *,
+        all: bool = False,
+        latest: bool = False,
+    ) -> None:
+        """Restarts one or more pods
+
+        Parameters:
+            x: One or more pods to restart
+            all: Restart all pods
+            latest: Restart the latest pod
+
+        # Raises
+            `python_on_whales.exceptions.NoSuchPod` if any pods does not exist.
+        """
+        pods = to_list(x)
+        if len(pods) == 0 and not latest and not all:
+            return
+
+        full_cmd = self.docker_cmd + ["pod", "restart"]
+        full_cmd.add_flag("--all", all)
+        full_cmd.add_flag("--latest", latest)
+        full_cmd.extend([str(p) for p in pods])
+
+        run(full_cmd)
+
+    def start(
+        self,
+        x: Union[ValidPod, Sequence[ValidPod]] = (),
+        /,
+        *,
+        all: bool = False,
+        latest: bool = False,
+    ) -> None:
+        """Starts one or more pods
+
+        Parameters:
+            x: One or more pods to start
+            all: Start all pods
+            latest: Start the latest pod
+
+        # Raises
+            `python_on_whales.exceptions.NoSuchPod` if any pods does not exist.
+        """
+        pods = to_list(x)
+        if len(pods) == 0 and not latest and not all:
+            return
+
+        full_cmd = self.docker_cmd + ["pod", "start"]
+        full_cmd.add_flag("--all", all)
+        full_cmd.add_flag("--latest", latest)
+        full_cmd.extend([str(p) for p in pods])
+
+        run(full_cmd)
+
+    def stats(
+        self,
+        x: Union[ValidPod, Sequence[ValidPod]] = (),
+        /,
+        *,
+        all: bool = False,
+        latest: bool = False,
+    ) -> List[PodStats]:
+        """Get pods resource usage statistics
+
+        The data unit is the byte.
+
+        Parameters:
+            x: One or a list of pods
+            all: Get the stats of all pods, not just running ones
+            latest: Get stats for the latest pod
+
+        # Returns
+            A `List[python_on_whales.PodStats]`.
+        """
+        raise NotImplementedError  # TODO: implement PodStats class
+
+        pods = to_list(x)
+        if len(pods) == 0 and not all and not latest:
+            return []
+
+        full_cmd = self.docker_cmd + [
+            "pod",
+            "stats",
+            "--format",
+            "{{json .}}",
+            "--no-stream",
+        ]
+        full_cmd.add_flag("--all", all)
+        full_cmd.add_flag("--latest", latest)
+
+        stats_output = run(full_cmd)
+        return [PodStats(json.loads(s)) for s in stats_output.splitlines()]
+
+    def stop(
+        self,
+        x: Union[ValidPod, Sequence[ValidPod]] = (),
+        /,
+        *,
+        all: bool = False,
+        latest: bool = False,
```

Comment:
Here too, only one argument must be used, I wonder how we can add this information in the signature.

Reply:
deferred